packages/amazonq/test/unit/amazonqGumby/transformationResultsHandler.test.ts

```diff
@@ -6,11 +6,19 @@ import assert from 'assert'
 import sinon from 'sinon'
 import os from 'os'
 import { DiffModel, AddedChangeNode, ModifiedChangeNode } from 'aws-core-vscode/codewhisperer/node'
+import { PatchInfo } from 'aws-core-vscode/codewhisperer'
```

Comment:
Should be imported from `model.ts`

Reply:
Having trouble seeing how to import from model.ts as its in the amazonq test directory instead of core. I was looking at other tests and when importing they seem to import from 'aws-core-vscode/' whether it be from 'test' or 'shared' or 'codewhisperer' etc. I know this might get autogenerated but in that file we do import everything from model. Maybe I am just missing something ...